<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-12T17:31:12 PKT" name="Default suite" finished-at="2024-10-12T17:31:12 PKT" duration-ms="193">
    <groups>
    </groups>
    <test started-at="2024-10-12T17:31:12 PKT" name="Default test" finished-at="2024-10-12T17:31:12 PKT" duration-ms="193">
      <class name="tests.CompleteDirectDeliveryShipmentTest">
        <test-method is-config="true" signature="setupSuite()[pri:0, instance:tests.CompleteDirectDeliveryShipmentTest@3571b748]" started-at="2024-10-12T17:28:51 PKT" name="setupSuite" finished-at="2024-10-12T17:31:11 PKT" duration-ms="140666" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Cypress tests failed. Cannot proceed with Appium tests.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Cypress tests failed. Cannot proceed with Appium tests.
at tests.CompleteDirectDeliveryShipmentTest.setupSuite(CompleteDirectDeliveryShipmentTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:358)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupSuite -->
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.CompleteDirectDeliveryShipmentTest@3571b748]" started-at="2024-10-12T17:31:12 PKT" name="setup" finished-at="2024-10-12T17:31:12 PKT" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="completeDirectDeliveryShipment()[pri:0, instance:tests.CompleteDirectDeliveryShipmentTest@3571b748]" started-at="2024-10-12T17:31:12 PKT" name="completeDirectDeliveryShipment" finished-at="2024-10-12T17:31:12 PKT" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Cypress tests failed. Cannot proceed with Appium tests.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Cypress tests failed. Cannot proceed with Appium tests.
at tests.CompleteDirectDeliveryShipmentTest.setupSuite(CompleteDirectDeliveryShipmentTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:358)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- completeDirectDeliveryShipment -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CompleteDirectDeliveryShipmentTest@3571b748]" started-at="2024-10-12T17:31:12 PKT" name="tearDown" finished-at="2024-10-12T17:31:12 PKT" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.CompleteDirectDeliveryShipmentTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
